src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/index.tsx

```diff
@@ -44,6 +44,7 @@ function CustomBottomTabNavigator({initialRouteName, children, screenOptions, ..
 
     const styles = useThemeStyles();
     const stateToRender = getStateToRender(state);
+    const selectedTab = stateToRender.routes.at(-1)?.name;
```

Comment:
NAB now that we removed the platform specific files, we could also remove the directory and rename this file to `BottomTabBar.tsx`